__tests__/__mocks__/request.js

```diff
@@ -47,7 +47,12 @@ class Request extends RealRequest {
   }
 }
 
-module.exports = function(params: Object): Request {
+module.exports = function(initialParams: Object, callback?: ?Function): Request {
+  const params = {
```

Comment:
As request module is being mocked here, this simulates the original module options normalization.